docroot/themes/contrib/civictheme/includes/system_branding.inc

```diff
@@ -39,14 +39,14 @@ function civictheme_preprocess_block__system_branding_block(&$variables) {
     $alt = theme_get_setting('components.logo.image_alt') ?? t('CivicTheme logo');
 
     if (!empty($desktop_logo)) {
-      $variables['logos']['desktop'] = [
+      $variables['logos']['desktop']['primary'] = [
```

Comment:
just an FYI - the Drupal ticket for will be adding a field to support the second logo